code/controllers/subsystem/job.dm

```diff
@@ -507,6 +507,14 @@ SUBSYSTEM_DEF(job)
 				newplayer.new_character = living_mob
 			else
 				M = living_mob
+
+		var/obj/item/modular_computer/RPDA = locate(/obj/item/modular_computer) in living_mob.GetAllContents()
+		if(istype(RPDA))
```

Comment:
Please move L513 to L542 and remove the rest of this

Reply:
moved it down. didn't realize that was the conflict when I pr'd it and just thought it was misinterpreting the whole RPDA chunk as new because i changed one line, but i had accidentally created the whole chunk again